### PR TITLE
Reintroduce logging via JBoss Logging 3

### LINKED ISSUE #14: Reintroduce logging
It'd be great to be able to reintroduce (currently commented out) logging somehow that has been commented out. I've had to temporarily hack it back in for my development -- annoying subtle bug that I couldn't figure out from debugging alone -- and was clear once logging was back in. 

Also, I think the logging will add a lot of value when support cases start arriving, etc. 

What are people's thoughts about how best to do this?

### PR DESCRIPTION
**Pull request by msavy**
Fixes #14

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -44,7 +44,7 @@
         <version.org.jboss.arquillian>1.1.13.Final</version.org.jboss.arquillian>
         <version.org.jboss.arquillian.container.weld-embedded>2.0.0.Beta5</version.org.jboss.arquillian.container.weld-embedded>
         <version.org.jboss.jandex>2.0.5.Final</version.org.jboss.jandex>
-        <version.org.jboss.logging>2.0.1.Final</version.org.jboss.logging>
+        <version.org.jboss.logging>3.3.2.Final</version.org.jboss.logging>
```
Comment (kenfinnigan) [inline pom.xml]: Can this be changed to 3.3.1.Final as I think that's the version used in WF 11+ right now

Comment (msavy) [reply pom.xml]: @kenfinnigan Did a force push 🔨

**Comment by msavy** [comment]
@kenfinnigan - spoken with @jamezp and he indicated he's doing a PR in WildFly to update to 3.3.2.Final - (https://github.com/smallrye/smallrye-open-api/issues/14#issuecomment-408921942)

Happy to go with your call on it!

**Comment by kenfinnigan** [comment]
I'd stick with 3.3.1 for now as that's what's in versions already out there

**Comment by msavy** [comment]
Sounds good to me. Thanks!

**Pull request merged**